lib/l10n/app_fi.arb

```diff
@@ -1503,33 +1503,33 @@
       }
     }
   },
-  "qr_code_dialog_synchronizing": "Synchronizing to the network",
-  "qr_code_dialog_invoice": "Invoice",
-  "qr_code_dialog_action_close": "CLOSE",
-  "qr_code_dialog_share": "Share invoice data",
-  "qr_code_dialog_copy": "Copy invoice data",
-  "qr_code_dialog_copied": "Invoice data was copied to your clipboard.",
-  "qr_code_dialog_warning_message_error": "Failed to create invoice",
-  "qr_code_dialog_warning_message": "Keep Breez open until the payment is completed.",
-  "qr_code_dialog_warning_message_with_lsp": "A setup fee of {setupFee} ({fiatFee}) is applied to this invoice. Keep Breez open until the payment is completed.",
+  "qr_code_dialog_synchronizing": "Sovellus synkronoi tietoja",
+  "qr_code_dialog_invoice": "Lasku",
+  "qr_code_dialog_action_close": "SULJE",
+  "qr_code_dialog_share": "Jaa laskun tiedot",
+  "qr_code_dialog_copy": "Kopioi laskun tiedot",
+  "qr_code_dialog_copied": "Laskun tiedot kopioitiin leikepöydälle.",
+  "qr_code_dialog_warning_message_error": "Laskun luonti epäonnistui",
+  "qr_code_dialog_warning_message": "Älä sulje sovellusta ennenkuin tapahtuma on valmis.",
+  "qr_code_dialog_warning_message_with_lsp": "Tämän laskun suorituksesta veloitetaan saapuvan kanavan perustamiskulu {setupFee} eli ({fiat}). Älä sulje sovellusta ennenkuin tapahtuma valmistuu.",
```

Comment:
You replaced the `fiatFee` key by `fiat`m it is necessary to use the `fiatFee` key.

Reply:
Fixed